CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 3.4.0 (2019-09-24)
+
+- Added the single file attachment component. [#60](https://github.com/blackbaud/skyux-forms/pull/60)
+
```

Comment:
We usually don't leave space between lines in changelogs. Is there a reason to do it here?

Reply:
I went ahead and deleted the extra lines while making a couple other small doc tweaks.